python/examples/integration/integration_client.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import print_function
+
+import time
+import argparse
+
+import grpc
+from jaeger_client import Config
```

Comment:
I added a requirements.txt file in the example directory that includes jaeger-client as an dependency.